test/helper/ui.js

```diff
@@ -179,6 +179,9 @@ export const hasOneSocialBigButton = hasOneViewFn(
   '.auth0-lock-social-button.auth0-lock-social-big-button'
 );
 export const hasPasswordInput = hasInputFn('password');
```

Comment:
How is this used? Should it account for the input being hidden?

Reply:
Maybe that's handled by the `hasHiddenPasswordInput` you added but still worth a consideration.

Reply:
not sure what you mean 😬